public/500.html

```diff
@@ -1,25 +1,87 @@
 <!DOCTYPE html>
 <html>
-<head>
-  <title>We're sorry, but something went wrong (500)</title>
-  <style type="text/css">
-    body { background-color: #fff; color: #666; text-align: center; font-family: arial, sans-serif; }
-    div.dialog {
-      width: 25em;
-      padding: 0 4em;
-      margin: 4em auto 0 auto;
-      border: 1px solid #ccc;
-      border-right-color: #999;
-      border-bottom-color: #999;
-    }
-    h1 { font-size: 100%; color: #f00; line-height: 1.5em; }
-  </style>
-</head>
+  <head>
+    <title>We're sorry, but something went wrong (500)</title>
+    <style type="text/css">
```

Comment:
Just a few styling tweaks:
* Making the heading bigger and a darker shade of gray
* making the description slightly bigger and with a lighter shade of gray
* It's a little off center vertically - we can get it perfect by:
  *  giving full height to the html and body elements,
  * Removing the vertical padding
  * Giving main a full min height. 
* With most of the padding gone we can remove the media queries.

It's easier to paste it all in here instead of going through them one by one:
```html
<!DOCTYPE html>
<html>
  <head>
    <title> Server error (500) | The Odin Project</title>
    <style type="text/css">
      html {
        height: 100%;
      }
      body {
        text-align: center;
        font-family: Arial, Helvetica, sans-serif;
        height: 100%;
        margin: 0;
      }
      main {
        display: grid;
        place-items: center;
        padding: 0rem 3rem;
        min-height: 100%;
      }
      h1 {
        margin-top: 1rem;
        font-size: 2.5rem;
        color: #111827;
        line-height: 2.25rem;
        font-weight: 700;
      }
      p {
        font-size: 1.125rem;
      }
      .description {
        line-height: 1.75rem;
        margin-top: 1.5rem;
        color: rgb(75, 85, 99);
      }
      .status {
        font-size: 1.1rem;
      }
      .highlight {
        color: rgb(169 121 43);
      }
      .button {
        border-radius: 0.375rem;
        background-color: rgb(15 118 110);
        padding: 0.875rem 0.625rem;
        color: white;
        outline-style: solid;
      }
      .button:hover {
        background-color: rgb(17 94 89);
      }
      .button:focus-visible {
        outline-style: solid;
        outline-width: 2px;
        outline-offset: 2px;
        outline-color: rgb(17 94 89);
      }
      div.container {
        margin-top: 2.5rem;
        display: flex;
        align-items: center;
        justify-content: center;
        column-gap: 1.5rem;
      }
      a {
        text-decoration: none;
      }
    </style>
  </head>

  <body>
    <!-- This file lives in public/422.html -->
    <main>
      <div>
        <p class="highlight status">500</p>
        <h1>Server error</h1>
        <p class="description">We're sorry, but something went wrong.</p>
        <div class="container">
          <a href="/home" class="button">Go back home</a>
          <a href="https://github.com/TheOdinProject/theodinproject/issues/new/choose" class="highlight">Report an issue on GitHub <span aria-hidden="true">&rarr;</span></a>
        </div>
      </div>
    </main>
  </body>
</html>
``` 

This is what it looks like after all these changes:

<img width="1645" alt="Screenshot 2023-06-25 at 17 04 39" src="https://github.com/TheOdinProject/theodinproject/assets/7963776/e5fcb839-27ef-4330-8ea5-316031a7d832">

We'll also need to make the same changes to the 422 file.

Reply:
I mostly like these styling changes, but I'm not entirely sold on 
```css
.status {
        font-size: 1.1rem;
      }
```

That makes the status almost as big as the actual text content (1.1rem vs 1.125rem), and I feel like that makes it distract and draw the attention too much for something thats ultimately not really supposed to be that interesting for most people

Reply:
Whoops, that was a mistake. It should be the same size as the description content.

If it's smaller, it throws off the hierarchy/uniformity. It's a small difference, but noticeable and draws the eye more because there is a difference imo. It's just generally really difficult to pull off three different font sizes in close proximity.

But, I do see what you mean with what draws the eye, to offset that we can increase the size of the h1 to 3rem like they have it on the Tailwind component. And then both the description text and status can go down to 1rem. I think that produces something better with a very clear hierarchy:

<img width="1542" alt="Screenshot 2023-06-26 at 11 59 57" src="https://github.com/TheOdinProject/theodinproject/assets/7963776/b10169bf-612a-47a3-8341-45edf85b2f56">